plutus-core/cost-model/create-cost-model/BuiltinMemoryModels.hs

```diff
@@ -169,7 +169,7 @@ builtinMemoryModels = BuiltinCostModelBase
   , paramCountSetBits                    = Id $ ModelOneArgumentConstantCost 1
   , paramFindFirstSetBit                 = Id $ ModelOneArgumentConstantCost 1
   , paramRipemd_160                      = Id $ hashMemModel Hash.ripemd_160
-  , paramExpModInteger                   = Id $ ModelThreeArgumentsConstantCost 100000000000 -- FIXME: stub
+  , paramExpModInteger                   = Id $ ModelThreeArgumentsLinearInZ identityFunction
```

Comment:
Oh wait!  This is the size in bytes, not words, so it'll be 8 times bigger than it should be.  The size of the result will be the same as the size of the modulus, but we're using a non-standard measure here.